PostFreeSurfer/PostFreeSurferPipeline.sh

```diff
@@ -112,6 +160,27 @@ log_Check_Env_Var FSLDIR
 HCPPIPEDIR_PostFS="$HCPPIPEDIR/PostFreeSurfer/scripts"
 PipelineScripts="$HCPPIPEDIR_PostFS"
 
+#ExperimentRoot points to actual experiment directory
+case "$LongitudinalMode" in
+    NONE)       
+        ExperimentRoot=$Session
+        IsLongitudinal=0
+        ;;
+    TIMEPOINT_STAGE1|TIMEPOINT_STAGE2) 
+        ExperimentRoot="$Session".long."$LongitudinalTemplate"
+        IsLongitudinal=1 
+        ;;
+    TEMPLATE)
+        ExperimentRoot="$Subject".long."$LongitudinalTemplate"
```

Comment:
Please add a comment drawing attention to the small difference between the ExperimentRoot settings, as I had in my suggestion.

Reply:
Ok